cloudquery/sdk/plugin/plugin.py

```diff
@@ -29,10 +29,32 @@ class SyncOptions:
     backend_options: BackendOptions = None
 
 
+@dataclass
+class BuildTarget:
+    os: str = None
+    arch: str = None
+
+
+@dataclass
+class Options:
+    dockerfile: str = None
+    build_targets: List[BuildTarget] = None
+    team: str = None
+    kind: str = None
+
+
 class Plugin:
-    def __init__(self, name: str, version: str) -> None:
+    def __init__(self, name: str, version: str, opts: Options = None) -> None:
         self._name = name
         self._version = version
+        self._opts = Options() if opts is None else opts
+        if self._opts.dockerfile is None:
+            self._opts.dockerfile = "Dockerfile"
+        if self._opts.build_targets is None:
+            self._opts.build_targets = [
+                BuildTarget("linux", "amd64"),
```

Comment:
This matches our current supported platforms https://github.com/cloudquery/cloudquery/blob/d76f083b5027efd579d1432f59c1d8955182d0a1/.github/workflows/release_plugin.yml#L93

Reply:
Do we want Darwin targets as defaults? A lot of people do development on Macs (including us).

Reply:
I believe you can still run Linux images on MacOS. For example Node.js images has only Linux platform:
https://hub.docker.com/_/node/tags?page=1
![image](https://github.com/cloudquery/plugin-sdk-python/assets/26760571/8cbc7e0d-3e3a-40e1-8627-6be62bcbc4e3)